src/Controller/ModulesController.php

```diff
@@ -334,4 +365,104 @@ public function upload()
             'id' => $response['data']['id'],
         ]);
     }
+
+    /**
+     * Export data in csv format
+     *
+     * @return \Cake\Http\Response
+     */
+    public function changeStatus() : Response
+    {
+        $this->request->allowMethod(['post']);
+        $ids = $this->request->getData('ids');
+        $status = $this->request->getData('status');
+        if (!empty($ids)) { // export selected (filter by id)
+            $ids = explode(',', $ids);
+            foreach ($ids as $id) {
+                $data = [
+                    'id' => $id,
+                    'status' => $status,
+                ];
+                $this->apiClient->saveObject($this->objectType, $data);
+            }
+        }
+
+        return $this->redirect(['_name' => 'modules:list', 'object_type' => $this->objectType]);
+    }
+
+    /**
+     * Export data in csv format
+     *
+     * @return void
+     */
+    public function export() : void
```

Comment:
Why not put this in an `ExportController`? Since we have already an `ImportController` it makes sense.